File: crates/target_csharp_system_text/output/geojson/GeojsonObjectLineString.cs

using System.Collections.Generic;

using System.Text.Json.Serialization;

namespace JtdCodegenE2E
{
    /// <summary>
    /// For type "LineString", the "coordinates" member is an array of two
    /// or more positions.
    /// </summary>

    public class GeojsonObjectLineString : GeojsonObject
    {
        [JsonPropertyName("type")]
        public string Type_ { get => "LineString"; }

        /// <summary>

        /// </summary>

        [JsonPropertyName("coordinates")]
        public IList<Position> Coordinates { get; set; }

    }
}

File: crates/target_java_jackson/output/geojson/GeojsonObjectLineString.java
package com.example;


import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

import com.fasterxml.jackson.annotation.JsonInclude;

import com.fasterxml.jackson.annotation.JsonProperty;

import com.fasterxml.jackson.databind.annotation.JsonSerialize;

import java.util.List;

/**
 * For type "LineString", the "coordinates" member is an array of two
 * or more positions.
 */

@JsonSerialize
@JsonIgnoreProperties(ignoreUnknown = true)
public class GeojsonObjectLineString extends GeojsonObject {

    
    @JsonProperty("coordinates")
    private List<Position> coordinates;

    @JsonInclude(JsonInclude.Include.NON_NULL)
    @JsonProperty("bbox")
    private BoundingBox bbox;


    public GeojsonObjectLineString() {
    }


    public List<Position> getCoordinates() {
        return this.coordinates;
    }

    public void setCoordinates(List<Position> coordinates) {
        this.coordinates = coordinates;
    }

    public BoundingBox getBbox() {
        return this.bbox;
    }

    public void setBbox(BoundingBox bbox) {
        this.bbox = bbox;
    }

}
